roles/testnode/tasks/apt/packages.yml

```diff
@@ -12,7 +12,9 @@
     name: "{{ packages_to_upgrade|list }}"
     state: latest
     force: yes
-  when: packages_to_upgrade|length > 0
+  when:
+  - packages_to_upgrade|length > 0
+  - not containerized_node
```

Comment:
Why can't we upgrade packages in containers?  What was the problem?

Reply:
Ah, this can be left out. The "problem" is that for the workflow I'm testing, it brings a <2min ansible run up to ~9min. But I think I can also just skip the packages tag.